source/Octopus.Tentacle/Services/KubernetesServiceAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Octopus.Tentacle.Services
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    public class KubernetesServiceAttribute : Attribute, IServiceAttribute
```

Comment:
The `KubernetesServiceAttribute` which allows us to conditionally register Kubernetes Services... I felt it was better to keep all the services being registered in the same module, rather than splitting them into separate modules